blogposts/remote-software-development-made-easier.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Remote software development made easier"
+author: Sourcegraph
```

Comment:
We haven't done this before and it strikes me as being more corporate when it doesn't come from an individual. 

Why not add your name, or Stephen's?

Reply:
There were so many people involved in producing this content that I didn't want to take sole credit for it, but I understand if stating Sourcegraph also gives a wrong impression...